### ISSUE TITLE
Issue with latin and cyrillic characters when using T2A

### ISSUE THREAD

**Issue by palkinev** [textext/textext#240]
**Description of the problem**
When \usepackage[T2A]{fontenc} is added to default_packages.tex Tex Text produces misshapen output

**Steps to reproduce the behavior (if not obvious from description)**:
1. I followed installation guide from https://textext.github.io/textext/install/linux.html
Inkscape 1.0 is installed as
apt install -t buster-backports inkscape
on debian 10

2. Added following lines to default_packages.tex
\usepackage[T2A]{fontenc}
\usepackage[utf8]{inputenc}
\usepackage[russian, english]{babel}

which is standard for cyrillic characters support in latex 
https://ru.overleaf.com/learn/latex/Russian

3. Inside Inkscape I created an empty document, then launched "Extensions - Text - Tex Text" 
and typed
"
hello world
"
Result is malformed. 
![Screenshot_2020-07-25_13-22-16](https://user-images.githubusercontent.com/31729290/88454763-61c73a80-ce7a-11ea-9ad2-8f41eec0a211.png)

Same thing happens if I write anything using cyrillic characters "Добрый день" (Good day) 
![Screenshot_2020-07-25_13-25-15](https://user-images.githubusercontent.com/31729290/88454757-5e33b380-ce7a-11ea-8c1c-61441eb498a0.png)

**System information**

- TexText version: 1.1.0
- Inkscape version: 1.0
- Operating system: debian 10, 64-bit
- Python 2.7.16

**Comment by sizmailov** [comment]
Hi, does it work with pdflatex on its own? Can you follow steps that textext performs manually and figure out when error appears? 

This looks like a bad configuration/misuse of latex packages to me.

**Comment by palkinev** [comment]
I created minimal .tex file following base.py in TexText:
```
\documentclass{article}
\usepackage{amsmath,amsthm,amssymb,amsfonts}
\usepackage{color}
\usepackage[T2A]{fontenc}
\usepackage[utf8]{inputenc}
\usepackage[russian, english]{babel}
\pagestyle{empty}
\begin{document}
hello world\\
Добрый день
\end{document}
```
output is fine:
![123](https://user-images.githubusercontent.com/31729290/88796444-4de94480-d1aa-11ea-9d83-5d404665857d.png)

Then I converted .pdf to .png via inkscape with:

inkscape 123.pdf --pdf-poppler --export-overwrite --export-type=png --export-text-to-path --export-area-drawing  --export-dpi=300

Which produces an familiar irregular output:
![123](https://user-images.githubusercontent.com/31729290/88796653-addfeb00-d1aa-11ea-9d33-76ada86a2a1e.png)

During execution last command also prints a warning:
"
** (org.inkscape.Inkscape:20444): WARNING **: <timecode>: Fonts dir '/usr/share/inkscape/fonts' does not exist and will be ignored.
"
I think that most likely it is unrelated to this issue and here is why:

I have another PC at home with Arch Linux installed.

I repeated the procedure described above and obtained normal output:
![123](https://user-images.githubusercontent.com/31729290/88930850-3db68f80-d2a6-11ea-9345-0f07227649c5.png)

which also means that TexText extension works properly in Inkscape.

And ```/usr/share/inkscape/fonts``` font dir is present, but empty in case of Arch linux, which suggests that warning above can
be neglected.

Then I took my laptop, which has fresh debian 10 installation and repeated this procedure again,
which surprisingly for me gave me a correct output while ```/usr/share/inkscape/fonts``` dir is absent.

This probably means that the debian system I originally used prior to posting this issue has some font issues.
But I have not figures this out yet.

**Comment by sizmailov** [comment]
I've repeated your steps on ubuntu 19.04 with inkscape 1.1-dev and it works as expected.
Seems to be a either an inkscape bug or/and font misconfiguration. 
Try to run `fc-cache` and make sure you have all required fonts installed (see [inkscape wiki](https://wiki.inkscape.org/wiki/index.php/Installing_fonts)) .
If it doesn't help try [fresher version of inkscape](https://gitlab.com/inkscape/inkscape/-/jobs/artifacts/master/download?job=appimage%3Alinux), just tried to convert pdf->svg.

**Comment by palkinev** [comment]
I am glad to report that issue can be now considered solved.

Latex logs indicated clearly that a debian 10 PC and laptop were loading different fonts (but both with cyrillic support ).

A debian system I originally used prior to posting this issue used **lh** font which is provided by
```
texlive-lang-cyrillic
```
latex package.

And another machine used different font - **cm-super**.


Turns out that solution was:
```
apt install cm-super
```
(I also removed ~/.texlive*)
